helix-rest/src/main/java/org/apache/helix/rest/server/json/instance/StoppableCheck.java

```diff
@@ -30,8 +30,8 @@
   // Category to differentiate which step the check fails
   public enum Category {
     HELIX_OWN_CHECK("Helix:"),
-    CUSTOM_INSTANCE_CHECK("CustomInstance:"),
-    CUSTOM_PARTITION_CHECK("CustomPartition:");
+    CUSTOM_INSTANCE_CHECK("CustomInstanceHealthFailure:"),
+    CUSTOM_PARTITION_CHECK("CustomPartitionHealthFailure:");
```

Comment:
There are other fields in the response json as below.
```
{
  "instance_stoppable_parallel" : [ ],
  "instance_not_stoppable_with_reasons" : {
    "localhost_8080": [
        "CustomPartition:partion1",
        "CustomPartition:partition2"
    ]
  }
}

{
  "instance_stoppable_parallel" : [ ],
  "instance_not_stoppable_with_reasons" : {
    "localhost_8080" : [ "CustomInstance:localhost_8080" ]
  }
}
```

 Shall we make the keys format consistent: `custom_health_failure_instance: `? (`_instance` at the end indicates the value is an instance. And add a space after colon `:`?) It looks more consistent and clearer to me.
```
{
  "instance_stoppable_parallel" : [ ],
  "instance_not_stoppable_with_reasons" : {
    "localhost_8080": [
        "custom_health_failure_partition: partion1",
        "custom_health_failure_partition: partition2"
    ]
  }
}
```

Just a thought: not sure if we have to consider backward-compatibility: if some tools are parsing the json response to determine stoppable, this change will break the tools. Internal code search doesn't show such code usage.

Reply:
Good suggestion.